### ISSUE TITLE
Forums sign up issue

### ISSUE THREAD

**Issue by ounos** [bolt/site-v20#14]
I can't sign up using my github account.
Although I provide my email address, I get the following error:  ```The email field is required.```

**Comment by SahAssar** [comment]
Yeah, just tried it with another github account. Same thing. I guess this is what we get for using beta software :D

**Comment by kehugter** [comment]
Any news? I get the email error with all registration options.

**Comment by bobdenotter** [comment]
Nope, still waiting on a Flarum update.. 

You can still register with an "old fashioned" username and password, though! :-)

**Comment by kehugter** [comment]
Didn't work either... will try with other domains later.
